### PR TITLE
feat: add group tab in admin section

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes https://github.com/fossasia/open-event-frontend/issues/6717
Follow up for https://github.com/fossasia/open-event-frontend/pull/7084

required https://github.com/fossasia/open-event-server/pull/7918 from server

Screenshot from 2021-04-28 07-58-40
Screenshot from 2021-04-28 07-58-37

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Review by MeghalBisht — APPROVED** [review]
Works!
Screenshot from 2021-05-07 00-14-11

File: app/templates/components/ui-table/cell/admin/groups/cell-group-name.hbs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<span>{{t @record}}</span>
+<div class="ui hidden divider"></div>
+<div class="ui horizontal compact basic buttons">
+  <LinkTo @route="groups.edit" @model={{@extraRecords.id}}>
+    <UiPopup @content={{t "Edit"}}  @class="ui icon button">
+      <i class="edit icon"></i>
+    </UiPopup>
+  </LinkTo>
+  {{#if (not @extraRecords.deletedAt)}}
+    <UiPopup @content={{t "Delete"}} @class="ui icon button" @click={{action @props.actions.deleteGroup @extraRecords.id}}>
```
Comment (MeghalBisht) [inline app/templates/components/ui-table/cell/admin/groups/cell-group-name.hbs:10]:
```suggestion
<UiPopup @content={{t "Delete"}} @class="ui icon button" @click={{action (confirm (t "Are you sure you would like to delete this group?") (action @props.actions.deleteGroup @extraRecords.id))}}>
```

File: app/routes/admin/groups/list.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      filterOptions = [
+        {
+          name : 'deleted-at',
+          op   : 'ne',
+          val  : null
+        }
+      ];
+    } else {
+      filterOptions = [];
+    }
```
Comment (MeghalBisht) [inline app/routes/admin/groups/list.js:39]:
```suggestion
    }
    filterOptions = this.applySearchFilters(filterOptions, params, searchField);
```

File: app/templates/components/ui-table/cell/admin/groups/cell-group-name.hbs (hunk 1 above)
Comment (maze-runnar) [reply app/templates/components/ui-table/cell/admin/groups/cell-group-name.hbs:10]: we don't add confirmation in admin section. I have removed that from admin event section in past.

**Review by MeghalBisht — DISMISSED** [review]
Works!

**Comment by MeghalBisht** [comment]
Why closed?

**Comment by maze-runnar** [comment]
> Why closed?

I deleted the branch by mistake.

**Review by sachinchauhan2889 — APPROVED** [review]

**Review by mariobehling — APPROVED** [review]
Looks good. Please change it to testing branch.

**Comment by ranjitkshah** [comment]
works!

**Pull request merged**